controllers/game.controller.js

```diff
@@ -47,15 +66,22 @@ export const updateGame = async (req, res, next) => {
       "message": "Game updated successfully.",
       "data": result.rows[0]
     });
-  } catch (error) {
+  } else {
+    return res.status(403).json({ message: 'Only admins can update game' });
+  }
+}  catch (error) {
     next(error);
   }
 };
 
+/*
+//TODO: Only allow admins to delete games
+*/
```

Comment:
Eliminate the `Todo` once the it has been done.

Reply:
Alright.